src/Libraries/Microsoft.Extensions.Http.AutoClient/Microsoft.Extensions.Http.AutoClient.json

```diff
@@ -115,6 +115,16 @@
         }
       ]
     },
+    {
```

Comment:
@geeknoid is that the correct way to change this file (manually)? or should I use a tool for correctness.

Reply:
You can change it manually, that's fine.

Normally, I use ./scripts/MakeApiBaselines.ps1 to do this for me.

Reply:
I tried again with ApiChief directly (this ps1 failed with some invalid paths) and it worked. I was previously using it on a code generation dll while it should have been use on the main assembly.
Also added the required `-o` argument to generate the json file.

Example:

```
PS C:\code\extensions\eng\Tools\ApiChief> dotnet run -- "C:\code\extensions\artifacts\bin\Microsoft.Extensions.Http.AutoClient\Debug\net8.0\Microsoft.Extensions.Http.AutoClient.dll" emit baseline -o C:\code\extensions\src\Libraries\Microsoft.Extensions.Http.AutoClient\Microsoft.Extensions.Http.AutoClient.json
```

FYI @joperezr since we were not sure it could output this json file too.